tools/rss/spdlog.BUILD

```diff
@@ -0,0 +1,9 @@
+cc_library(
```

Comment:
is this a rsslib requirement?

Reply:
Yes, it is a dependency of Rss
https://intel.github.io/ad-rss-lib/BUILDING/index.html